apps/python-service/tests/test_auth.py

```
@@ -0,0 +1,105 @@
+        "/test-endpoint",
+        content=payload,
+        headers={
+            "X-Timestamp": str(timestamp),
+            "X-Internal-Signature": signature
+        }
+    )
+    assert response.status_code == 200
+    assert response.json()["message"] == "success"
+
+def test_replay_attack(mock_redis):
+    mock_get, mock_set = mock_redis
```

Ruff (0.15.14)

[warning] Unpacked variable `mock_set` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)